manifests/i/ImageMagick/ImageMagick/7.1.1.20/ImageMagick.ImageMagick.installer.yaml

```diff
@@ -0,0 +1,41 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=QUSU.CRLF.7-4-0-preview-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ImageMagick.ImageMagick
+PackageVersion: 7.1.1.20
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
```

Comment:
the installer would benefit from an installerSwitches!

The installer constantly asks whether it shall overwrite existing files, which could easily be fixed using the cmd-line arg: "/SUPPRESSMSGBOXES" in a installerSwitch